services/src/api/model/responses.rs

```diff
@@ -0,0 +1,169 @@
+use crate::handlers::ErrorResponse;
```

Comment:
general comment: maybe we should organize the errors differently? Maybe one sub-module for each handler? Or make every handler a folder with the functionality in one file and the errors in a second file?

What do you think?

Reply:
I think it is a great idea to have one module per handler.

Firstly, some handlers get quite long when you sum up the error definition, the openapi doc, the handler logic and maybe custom request types which are only used in that handler. When you have multiple handlers in one file it is not so easy to know where one handler starts or stops and harder to navigate.

Secondly, you would have the handler and the corresponding tests nearer together. This way you can look at the handler and the corresonding tests more easily.

Having a folder with a separate file for handler and error would create too much files in my opinion. I would like the idea that everything that has to do with one handler is kept together.

Reply:
[0c01c65](https://github.com/geo-engine/geoengine/pull/751/commits/0c01c6596789e181bd0fca1441fd5c7acfb71965) Like this? Look good.